### DISCUSSION TITLE
How can I use mucommander in Windows without the executable

### DISCUSSION THREAD

**Discussion by pandit-tech** [mucommander/mucommander#1300] [General]
Dear everybody

I work with a app-blocked Windows-10 client. So I can't run any executable program's besides a few allowed one, like Office. Fortunately I can run JAVA programs, I wrote some simple ones for my need. I like a good file manager, and Windows Explorer isn't. 

My question is how I can run MUcommander on windows without the executable. I tried a lot, but the results are a lot of errors. The main executable is a black-box to me, It's a kind of wrapper with additional commands I think. , I like to know what steps happens before the JAR file is executed. I hope someone can help me,.

Thanks in advance

**Comment by pskowronek** [comment]
Hi, what type of muC distribution have you tried so far?
In portable version you can see `mucommander.sh` that you could convert into .bat quite easily. Do you have linux subsystem (WSL) on that Win10 so you could try .sh directly?

**Comment by ahadas** [comment]
@pandit-tech in addition to @pskowronek 's comment above, it's fairly easy to run muCommander from the command line using the main jar (in version 1.5.2, it's `mucommander-1.5.2.jar`). for instance, you can run the application that is installed using the msi installer with: `java -cp "c:\Program Files\muCommander\app\mucommander-1.5.2.jar" com.mucommander.main.muCommander`. it should be similar with the portable version
